src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -156,21 +154,14 @@ public void sendBuildMessage(StepParameters stepParameters) {
 
         String webhookUrl = stepParameters.getWebhookUrl();
         try {
-            if (webhookUrl != null) {
```

Comment:
Well, I missed this. You now requires me do use properties
```
node {
properties([[$class: 'WebhookJobProperty', webhooks: [[name: 'webhook', notifyAborted: false, notifyBackToNormal: false, notifyFailure: false, notifyNotBuilt: false, notifyRepeatedFailure: false, notifySuccess: false, notifyUnstable: false, startNotification: false, timeout: 30000, url: 'https://webhook.com']]], pipelineTriggers([[$class: 'PeriodicFolderTrigger', interval: '1d']])])
office365ConnectorSend
}
```

Where before I could settle for
```
node {
  withCredentials([string(credentialsId: 'webhook', variable: 'webhook')]) {
      office365ConnectorSend webhookUrl: "${env.webhook}"
   }
}
```

Reply:
I'll fix it on my #22